test/index.spec.js

```diff
@@ -39,6 +36,18 @@ async function spawnNode (boostrap = []) {
   }
 }
 
+async function waitForPeer (ipfs, remote) {
+  for (let i = 0; i < 5; i++) {
+    const peers = await ipfs.swarm.peers()
+
+    if (peers.find(({ peer }) => peer === remote.id)) {
+      return
+    }
+
+    await delay(1000)
+  }
+}
```

Comment:
Using something like `p-wait-for` would be better than delay here to avoid waiting unnecessarily long.

Reply:
I've removed this as the problem was with node 15